jekyll-seo-tag.gemspec

```diff
@@ -30,5 +30,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.15"
   spec.add_development_dependency "html-proofer", "~> 3.7"
   spec.add_development_dependency "rspec", "~> 3.5"
-  spec.add_development_dependency "rubocop", "0.51"
+  spec.add_development_dependency "rubocop", "0.56.0"
```

Comment:
Do we want this to be `"~> 0.56.0"`?

Reply:
yes, because Rubocop has a tendency to add new cops that make our tests fail :)

Reply:
In patch releases?

Right now, it looks like you've locked it to `"0.56.0"`. Should we change it to `"~> 0.56.0"` to allow (ie) `"0.56.1"`?